edb/pgsql/metaschema.py

```diff
@@ -387,6 +387,30 @@ def __init__(self) -> None:
         )
 
 
+class MultirangeToJsonFunction(dbops.Function):
+    """Convert anymultirange to a jsonb object."""
+    text = r'''
```

Comment:
This does the wrong thing when the input is `NULL`... and so does the existing `range_to_jsonb` function.

We have a test that I wrote that tries to test casts from NULL for every cast we have... but it doesn't handle the polymorphic casts, so it missed this.

I'll go fix that

Reply:
I've updated the SQL to include a `NULL` check just like for ranges.